subworkflows/local/prepare_genome.nf

```diff
@@ -45,6 +47,14 @@ workflow PREPARE_GENOME {
     TABIX_KNOWN_INDELS(known_indels.map{ it -> [[id:it[0].baseName], it] })
     TABIX_PON(pon.map{ it -> [[id:it[0].baseName], it] })
 
+    chr_files = chr_dir
+    //TODO this works, but is not pretty. I will leave this in yuor hands during refactoring @Maxime
+    if ( params.chr_dir.endsWith('tar.gz')){
+        UNTAR_CHR_DIR(chr_dir.map{ it -> [[id:it[0].baseName], it] })
+        chr_files = UNTAR_CHR_DIR.out.untar.map{ it[1] }
+        ch_versions = ch_versions.mix(UNTAR_CHR_DIR.out.versions)
+    }
+
     // Gather versions of all tools used
     ch_versions  = ch_versions.mix(SAMTOOLS_FAIDX.out.versions)
     ch_versions = ch_versions.mix(BWAMEM1_INDEX.out.versions)
```

Comment:
Can you fix the alignment here?
Definitively my fault, I should not have let this slide...